src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1342,3 +1342,93 @@ def validate_or_policy(namespace):
 
         if "policyId" in or_policy.keys() and or_policy["policyId"]:
             namespace.policy_id = or_policy['policyId']
+
+
+def get_url_with_sas(cmd, namespace, url=None, container=None, blob=None, share=None, file_path=None):
```

Comment:
Can this method be reused in other places? If so, maybe we can consider putting it into the utils~